yt_dlp/extractor/biqle.py

```diff
@@ -47,56 +46,51 @@ def _real_extract(self, url):
         video_id = self._match_id(url)
         webpage = self._download_webpage(url, video_id)
 
-        title = self._search_regex(
-            r'<meta.*itemprop ?= ?"name".*content ?= ?"([^"]+)".*/>',
-            webpage, 'Title', default='Empty Title', fatal=False)
-        uploadDate = self._search_regex(
-            r'<meta.*itemprop ?= ?"uploadDate".*content ?= ?"([^"]+)".*/?>',
-            webpage, 'Upload Date', fatal=False)
-        timestamp = unified_timestamp(uploadDate)
-        description = self._search_regex(
-            r'<meta.*itemprop ?= ?"description".*content ?= ?"([^"]+)".*/>',
-            webpage, 'Description', fatal=False)
+        title = self._html_search_meta('name', webpage, 'Title', default=None, fatal=False)
+        upload_date = self._html_search_meta('uploadDate', webpage, 'Upload Date', default=None, fatal=False)
+        if upload_date is not None:
+            timestamp = unified_timestamp(upload_date)
+        description = self._html_search_meta('description', webpage, 'Description', default=None, fatal=False)
 
-        globalEmbed_url = self._search_regex(
-            r'<script.+?window.globEmbedUrl = \'((?:https?:)?//(?:daxab\.com|dxb\.to|[^/]+/player)/[^\']+)\'.*?></script>',
+        global_embed_url = self._search_regex(
+            r'<script.+?window.globEmbedUrl\s*=\s*\'((?:https?:)?//(?:daxab\.com|dxb\.to|[^/]+/player)/[^\']+)\'',
```

Comment:
```suggestion
            r'<script[^<]+?window.globEmbedUrl\s*=\s*\'((?:https?:)?//(?:daxab\.com|dxb\.to|[^/]+/player)/[^\']+)\'',
```
I think this is causing the need for dotall. In general, we want to avoid `.` completely in order to prevent issues like catastrophic backtracking and poor performance on large pages. So needing dotall often means there is some issue in the regex